conf/messages.en

```diff
@@ -65,3 +65,13 @@ upload.whatDoINeedToInclude = What information do I need to include in this file
 upload.acceptedFileType = Accepted file type: {0}
 upload.maxFileSize = Maximum file size: {0}
 upload.uploadAFile = Upload a file
+
+whatYouWillNeed.title = What you will need - Managing Pension Schemes - GOV.UK
```

Comment:
`Managing Pension Schemes - GOV.UK` is added automatically in non-sipp. 
https://github.com/hmrc/pension-scheme-return-frontend/blob/main/conf/messages.en#L101
I guess we should have the same?

Reply:
Fixed